.travis.yml

```diff
@@ -1,5 +1,10 @@
 ---
 language: ruby
-rvm: 2.1.2
-before_script: bundle exec appraisal install
-script: bundle exec appraisal rspec
+rvm:
+  - 2.1.8
```

Comment:
I missed the boat on this, but is there any reason to not just specify `2.1`, `2.2` and `2.3`, which should give us the latest patch version of each?